src/reports/components/outcome.scss

```diff
@@ -60,6 +60,11 @@ $outcome-count-border-size: 1.5px;
         font-weight: 700;
         margin-left: -8px;
         height: $outcome-chip-icon-size - (2 * $outcome-count-border-size);
+
+        @media screen and (forced-colors: active) {
+            padding-top: 2px;
+            padding-bottom: 1px;
```

Comment:
@DaveTryon Something about the bigger border bugged me, so I banged my head against this for a while and I think I have something that will remove the clipping but not require extra padding:

```suggestion
            forced-color-adjust: none;
            border-color: CanvasText !important;
            color: CanvasText;
```

In looking at this, I thought that the problem was the background color was being overridden from transparent to the canvas color, so I tried disabling forced colors here and then specifying the [high contrast colors](https://www.w3.org/TR/css-color-4/#css-system-colors) explicitly. Per [an Edge blog post](https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/#:~:text=New%2C%20standard%20system%20colors%20replace%20the%20old%20values%3A,Highlight%20..%20...%20%202%20more%20rows%20) I found later, this is not quite the full story; high contrast mode turns on text backplates (see heading about 60% down the page) and setting `forced-color-adjust: none` turns these off.

I think the `!important` that sets the border color may be avoidable by applying it directly to the `.outcome-chip.outcome-chip-fail .count` and other flavors when `forced-colors` is `active`.

Reply:
Thanks! I greatly prefer this solution. I left the !important on the .count, since it seems to bring the changes together in a more cohesive way that spreading it around.